meteor-app/imports/api/fake-data.js

```diff
@@ -2,21 +2,31 @@ import faker from 'faker'
 import { JOB_STATUS_READABLE } from '/imports/api/constants'
 
 export const fakeJob = () => {
-  // faker.seed(123)
+
+  // search items
   const name = faker.name.findName()
   const make = faker.random.arrayElement(['Apollo', 'Giant', 'Malvern Star', 'Specialized'])
   const color = faker.commerce.color()
+  const email = faker.internet.email()
+  const phone = faker.phone.phoneNumberFormat()
+  const model = faker.random.arrayElement(['TZ1000', 'Fireball', 'Chicken Dance', 'Zoom2000']) 
```

Comment:
Haha, a bike called a 'Chicken Dance'? Better choices would be 'Trail Breaker', 'Samurai', 'Katana', 'Yukon', 'Cypress', 'Defy', 'CX1'